src/components/pages/AddFarmer/AddFarmer.jsx

```diff
@@ -0,0 +1,251 @@
+/** @format */
+
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+export class AddFarmer extends Component {
+  state = {
+    farmer: {},
+  }
+  render() {
+    return (
+      <div>
+        <header>
+          <div>Logo</div>
+          <div>
+            <button>Logout</button>
+          </div>
+        </header>
+
+        <section>
+          <button>Back</button>
+          <hr />
+
+          <form action="">
+            <div>
+              <h1>Personnel Information</h1>
+              <div>
+                <label for="">
```

Comment:
If you enclose the element you want to label 
you can leave the id="" out.

Reply:
its just a draft pr